composer.json

```diff
@@ -41,7 +41,7 @@
 
     "extra": {
         "branch-alias": {
-            "dev-master": "4.4-dev"
+            "dev-master": "4.8-dev"
```

Comment:
I suggest using `4.x-dev`, which is less precise about which minor version will be the next one, but avoids forgetting to update it at each release.

Reply:
updated, feel free to squash the changes before merge

Reply:
updated